result/dhcpv6interface.yaml

```diff
@@ -25,27 +25,38 @@ components:
             The name of a DHCPv6 Client.
           type: string
           x-field-uid: 1
-        ipv6_iapd_address:
+        ipv6_iapd_addresses:
```

Comment:
I think we need to have array of iapd each with address + prefix length and similarly array of addresses with address + gateway

Reply:
updated